pontoon/base/static/js/translate.js

```diff
@@ -1926,12 +1936,22 @@ var Pontoon = (function (my) {
           return;
         }
 
-        var source = $(this).find('.translation-clipboard').text();
+        // FTL Editor
+        if ($('#ftl').is('.active')) {
```

Comment:
I think that at some point we'll have to split implementation of editors into separate classes. That's not a blocker, just an observation.